gear/gear/http_server_utils.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Callable, Optional
+import orjson
+from aiohttp import web
+
+
+async def json_request(request: web.Request) -> Any:
+    return orjson.loads(await request.read())
```

Comment:
does `await request.json(loads=orjson.loads)` not work?

Reply:
The types don't match. `orjson` works with UTF-8 `bytes` not Python `str`. The [definition of `.json`](https://github.com/aio-libs/aiohttp/blob/b2a7983aed047e36c3418e788096d7f46cea7e1e/aiohttp/web_request.py#L666-L682) calls `.text` which unnecessarily creates a Python `str`. I couldn't figure out a way to avoid the allocation of the string without writing `json_request` myself.

Reply:
Ah lame, ok.